### DISCUSSION TITLE
Is Redux better than context?

### DISCUSSION THREAD

**Discussion by ra0321** [community/community#37002] [Apps, API and Webhooks] [answered]
Is Redux better than context?

**Comment by jupiterwebdev** [comment, marked as answer]
Linking to code
On GitHub.com, navigate to the main page of the repository.
Locate the code you'd like to link to: ...
Choose whether to select a single line or a range: ...
To the left of the line or range of lines, click . ...
Navigate to the conversation where you want to link to the code snippet.
